### ISSUE TITLE
The AnimatedChevronRightDownSmallVisualSource and AnimatedChevronUpDownSmallVisualSource of the AnimatedIcon cannot play correctly

### ISSUE THREAD

**Issue by SQc04** [microsoft/microsoft-ui-xaml#10243]
### Describe the bug

The AnimatedChevronRightDownSmallVisualSource and AnimatedChevronUpDownSmallVisualSource of the AnimatedIcon cannot play correctly.

### Steps to reproduce the bug

Go to WinUI3 Gallery > Styles > AnimatedIcon
Click Kind Combobox > AnimatedChevronUpDownSmallVisualSource or AnimatedChevronRightDownSmallVisualSource
Click Button

### Expected behavior

"When the kind is set to AnimatedChevronRightDownSmallVisualSource or AnimatedChevronUpDownSmallVisualSource, the icon should rotate.

### Screenshots

![Image](https://github.com/user-attachments/assets/c7849707-4d3d-4b46-9b3d-06e99e8d350a)
![Image](https://github.com/user-attachments/assets/3728584e-45a7-4b27-b35a-525e1dd9e64a)

### NuGet package version

WinUI 3 - Windows App SDK 1.6.3: 1.6.241114003

### Windows version

Windows Insider Build (xxxxx)

**Comment by karkarl** [comment]
Do you have animation settings turned on?

**Comment by SQc04** [comment]
> Do you have animation settings turned on?

Yes, the animation settings are turned on, only AnimatedChevronRightDownSmallVisualSource and AnimatedChevronUpDownSmallVisualSource cannot play.
![Image](https://github.com/user-attachments/assets/28385731-e0c2-4ba3-8444-e4a94fe8af39)
https://github.com/user-attachments/assets/a8f784bc-2fa5-4c72-8d5e-2acbedd53475

**Comment by Starcloudsea** [comment]
This issue recurs normally in my computer and still recurs in my apps and WinUI 3 Gallery, it looks like this issue has been on the back burner for a long time, considering fixing this?
